starter-code/js/index.js

```diff
@@ -1,11 +1,37 @@
-function deleteItem(e){
+console.log("index.js CONNECTED");
 
+var totalPriceItem = document.querySelector(".btn-success");
+var prixItem = document.querySelector(".prix-produit");
+var quantityProduct = document.querySelector("input");
+var newPrice = document.querySelector(".total-unite");
+
+
+
+// ITERATION 1
+totalPriceItem.onclick = function() {;
+newPrice.innerHTML =  "$" + parseFloat(prixItem.innerHTML) * parseFloat(quantityProduct.value);
```

Comment:
Cette méthode fonctionne très bien, mais elle est en effet un peu limitée s'il vient à y avoir plus de produits.
L'idée donc pour les itérations suivantes, c'est de fonctionner sur un principe de loop. Un exemple de ce à quoi ça pourrait ressembler, avec quelques explications :

```javascript
calculateBtn.onclick = function() {
  
  // Ici on va créer les variables nécessaires, qui seront chacune des arrays d'éléments DOM (sauf le globalPrice, qui ne sera qu'un bouton
  var totalPrices = document.querySelectorAll(".total-price");
  var quantities = document.querySelectorAll(".unit-quantity");
  var unitPrices = document.querySelectorAll(".unit-price");
  var globalPrice = document.querySelector(".global-price");

  // Ici on crée un counter, qui viendra nous aider à calculer le prix global
  var counter = 0;
  
  // On va looper travers nos lignes
  for( var i=0; i<unitPrices.length; i++ ) {

    // A chaque loop, on attribue à totalPrice la valeur de la multiplication du prix et de la quantité
    totalPrices[i].innerHTML = Number(quantities[i].value) * Number(unitPrices[i].innerHTML);

    // A chaque loop, on ajoute aussi le résultat de notre multiplication à notre counter
    counter += Number(totalPrices[i].innerHTML);
  }

  // Une fois les loops terminées, on donne à notre globalPrice la valeur de notre counter
  globalPrice.innerHTML = counter;
  
};
```